styles.css

```diff
@@ -1,14 +1,14 @@
 div.wirisanswerfielddecoration.wirisembeddeddecoration.wirisincorrect,
 div.wirisanswerfielddecoration.wiriseditordecoration.wirisincorrect div.wiriseditorwrapper {
-    background-image: url([[pix:core|i/grade_incorrect]]) !important;
+    background-image: url([[pix:core|i/grade_incorrect]]);
 }
 
 div.wirisanswerfielddecoration.wirisembeddeddecoration.wiriscorrect,
 div.wirisanswerfielddecoration.wiriseditordecoration.wiriscorrect div.wiriseditorwrapper {
-    background-image: url([[pix:core|i/grade_correct]]) !important;
+    background-image: url([[pix:core|i/grade_correct]]);
 }
 
 div.wirisanswerfielddecoration.wirisembeddeddecoration.wirispartiallycorrect,
 div.wirisanswerfielddecoration.wiriseditordecoration.wirispartiallycorrect div.wiriseditorwrapper {
-    background-image: url([[pix:core|i/grade_partiallycorrect]]) !important;
+    background-image: url([[pix:core|i/grade_partiallycorrect]]);
```

Comment:
Do we know where this is used?

Reply:
I've tested it assuming that it's the partially correct icon and I saw no difference with and without the `!important` property. May I just ignore this standard?